server/plugin/authorization.go

```diff
@@ -5,26 +5,20 @@ package plugin
 
 import (
 	"strings"
+
+	"github.com/mattermost/mattermost-plugin-msoffice/server/utils/bot"
 )
 
 // isAuthorized returns true if the user is authorized to use the workflow plugin's admin-level APIs/commands.
```

Comment:
workflow plugin? :)

Reply:
sorry, what's the question? I most definitely copied many things from the workflow plugin.

Reply:
Ah, nm!